File: AnimalShelterAPI/AnimalShelterAPI/Models/Animal.cs
using System;

namespace AnimalShelterAPI.Models
{
    public class Animal : BaseEntity
    {
        public DateTime AdmissionDate { get; set; }
        public DateTime? Birthday { get; set; }
        public DateTime? MicrochipIntegrationDate { get; set; }
        public DateTime? VaccinationDate { get; set; }
        public string SpecialID { get; set; }
        public string AdmissionCity { get; set; }
        public string AdmissionRegion { get; set; }
        public AnimalType AnimalType { get; set; }
        public Gender Gender { get; set; }
        public FurType FurType { get; set; }
        public string FurColor { get; set; }
        public string SpecialTags { get; set; }
        public string HealthCondition { get; set; }
        public string AdmissionOrganisationContacts { get; set; }
        public string TransferOrganisationContacts { get; set; }
        public Status Status { get; set; }
        public DateTime? StatusDate { get; set; }
        public DateTime LastReminder { get; set; }
    }
}

File: WebApp/src/app/models/animal.ts
import {Fur} from '@app/models/fur';

export class Animal {
  id: number;
  specialID: string;
  admissionDate: Date;
  microchipIntegrationDate: Date;
  vaccinationDate: Date;
  admissionCity: string;
  admissionRegion: string;
  animalType: number;
  months: number;
  furType: number;
  furColor: string;
  specialTags: string;
  healthCondition: string;
  admissionOrganisationContacts: string;
  statusID: number;
  statusDate: Date;
}
